app/views/dashboard/associated_users/destroy.js.coffee

```diff
@@ -17,6 +17,19 @@
 # DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
 # INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
 # TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
+# Send the user back to dashboard if theyre a member and not an admin
```

Comment:
If we destroy the current user and they are not an admin, redirect them to dashboard home. If we destroy the current user but they are an admin, re-render the page with updated rights information. If we destroy any other user, refresh the bootstraptable